### PR TITLE
MES 4422: Cat A Mod1 MVP Test Report For Bypass

### PR DESCRIPTION
**Pull request by faggc**
+ Commented out broken fault cards to allow bypass temporarily

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by MarkPatt89 — APPROVED** [review]

**Review by rossfauldsbjss — APPROVED** [review]
Looks good.

File: src/pages/test-report/cat-a-mod1/test-report.cat-a-mod1.page.ts [hunk 1]
```diff
@@ -246,4 +250,41 @@ export class TestReportCatAMod1Page extends BasePageComponent {
...
+    }));
+    this.store$.dispatch(new AddDrivingFault({
+      competency: Competencies.followingDistance,
+      newFaultCount: 1,
+    }));
+    this.store$.dispatch(new AddDrivingFault({
+      competency: Competencies.moveOffSafety,
+      newFaultCount: 2,
+    }));
+    this.store$.dispatch(new SetActivityCode('1'));
```
Comment (ammarhaiderbjss) [inline src/pages/test-report/cat-a-mod1/test-report.cat-a-mod1.page.ts:268]: I think we're missing a call to calculate the result?

Comment (ammarhaiderbjss) [reply src/pages/test-report/cat-a-mod1/test-report.cat-a-mod1.page.ts:268]: See https://github.com/dvsa/mes-mobile-app/pull/901/

Comment (faggc) [reply src/pages/test-report/cat-a-mod1/test-report.cat-a-mod1.page.ts:268]: I didn't see it when I looked at this example:
https://github.com/dvsa/mes-mobile-app/pull/1068/files

Is it needed?

**Review by ammarhaiderbjss — APPROVED** [review]

**Pull request merged**